Akshat111111/NexTrade
# Updated Scroll Button Functionality

I have made the following updates to your project:

**1. Scroll Button Enhancement:**
   - Modified the scroll button functionality to trigger at 10px instead of 300px for improved usability and a smoother experience for users scrolling shorter distances.

**2. Missing `style` Tag Fix:**
   - Discovered that the `style` tag was missing from the webpage, causing certain styles not to be applied correctly. I have added the necessary `style` tag and ensured the styles are properly embedded.


These changes aim to enhance both functionality and aesthetics, ensuring a smoother user experience and fixing the styling issues.

Please review the changes and let me know if any further adjustments are required!

**How to Test:**
- Scroll down on the page to 10px and verify that the scroll button appears.
- Check that the updated styles are now applied correctly after the `style` tag addition.


**Before:**

![IMg1](https://github.com/user-attachments/assets/9ce5d4af-730d-4d6e-aae2-766befa9ccff)

**After:**

![IMG2](https://github.com/user-attachments/assets/b115c94f-f3e4-43ba-97a2-024e61ce0628)

![IMG3](https://github.com/user-attachments/assets/3e8a57cd-519c-4fa4-8f0c-1f06e7722b78)


**Please Review and Merge it** @Akshat111111 #54
